models/zebra_motion.py

```diff
@@ -0,0 +1,9 @@
+from google.appengine.ext import ndb
+
+
+class ZebraMotion(ndb.Model):
```

Comment:
any particular reason why this is its own db model rather than being inline with match (it probably doesn't matter much though)

Reply:
Keeping our data models light-er by splitting out different types of data seems fine to me. The idea being we won't always need Zebra data if we're querying for a match (we probably just want time and whatnot)

Reply:
Also, I'm worried about the QueryCache for EventMatches being > 1MB if we put it all in the same model.

Reply:
works for me (I just wanted to have the reason documented here)